common/tier4_debug_rviz_plugin/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>tier4_debug_rviz_plugin</name>
+  <version>0.1.0</version>
+  <description>The tier4_debug_rviz_plugin package</description>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
Is it ok?
The original license is BSD.
https://github.com/jsk-ros-pkg/jsk_visualization/blob/master/jsk_visualization/package.xml#L14

Reply:
I'll ask someone.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2055/commits/5ed38fd4497d05612ee2e9a7593b565f80c45d6a
DONE